### PR TITLE
Pixy2

### PR DESCRIPTION
**Pull request by Aradig26**
adds a pixy2 subsystem that takes data from networktables and adds it to advantageScope and allows access from other subsystems

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/drive/Drive.java [hunk 1]
```diff
@@ -212,4 +218,11 @@ public Translation2d[] getModuleTranslations() {
       new Translation2d(-DriveConstants.TRACK_WIDTH_X / 2.0, -DriveConstants.TRACK_WIDTH_Y / 2.0)
     };
   }
+
+  public Command driveToNote(Supplier<Double> angle) {
+    return Commands.defer(
+            () -> new StraightDriveToPose(0, 0, Units.degreesToRadians(angle.get()), this),
+            Set.of(this))
+        .andThen(() -> new ChassisSpeeds(0, 1, 0), this);
```
Comment (EQMoore) [inline src/main/java/frc/robot/subsystems/drive/Drive.java:226]: Remove this line

**Review by EQMoore — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/drive/Drive.java (hunk 1 above)
Comment (EQMoore) [reply src/main/java/frc/robot/subsystems/drive/Drive.java:226]: line 226

**Review by admlvntv — CHANGES_REQUESTED** [review]
move this but otherwise lgtm

**Review by admlvntv — APPROVED** [review]
lgtm, make sure nt code is actually correct tho

**Review by EQMoore — APPROVED** [review]
lgtm

**Pull request merged**